app/views/contributions/index.html.erb

```diff
@@ -0,0 +1,70 @@
+<div class="container py-8">
+  <h1 class="title">
+    <span class="title">Getting Started: Ways to Contribute</span>
+  </h1>
+
+  <article class="prose mt-6 mb-12">
+    <p>Welcome to RubyVideo.dev! 🌟</p>
+
+    <p>We are thrilled to have you here and appreciate your interest in making this resource the best it can be for the community. If you would like to lend a hand, there are various ways you can contribute.</p>
+
+    <p>This page serves as a friendly starting point for anyone looking to get involved but unsure where to begin. Here are some ways you can help:</p>
+
+    <ul>
+      <li>Assist in adding missing GitHub handles to speakers</li>
+      <li>Help identify speakers based on conference screenshots</li>
+      <li>Contribute by adding last names to speakers</li>
+      <li>Attach Slides to talks</li>
+      <li>Add information about new conferences</li>
+    </ul>
+  </article>
+
+  <div class="mt-6">
+    <h2 class="mb-4">Speakers without GitHub handles</h2>
+
+    <article class="prose mb-6">These speakers are currently missing a GitHub handle. By adding a GitHub handle to their profile, we can enhance their speaker profiles with an avatar and automatically retrieve additional information.</article>
+
+    <div id="speakers" class="grid sm:grid-cols-2 lg:grid-cols-3 xl:grid-cols-4 gap-x-8 lg:gap-x-12 gap-y-2 min-w-full">
+      <% @speakers_without_github.each do |speaker| %>
+        <%= link_to edit_speaker_path(speaker), id: dom_id(speaker), class: "flex justify-between" do %>
+          <div class="flex items-center hover:bg-gray-200 p-2 rounded-lg">
+            <span><%= speaker.name %></span>
+          </div>
```

Comment:
I would suggest to open it in a remote modal 

adding data: {turbo_frame: "modal"} should do it

Reply:
yes, nice! 🙌🏼